test/solidity/Security/LiFiTimelockController.t.sol

```diff
@@ -66,6 +69,88 @@ contract LiFiTimelockControllerTest is Test {
         mockDiamond.transferOwnership(address(timelock));
     }
 
+    function testRevert_WhenInvalidConfig() public {
+        // Setup proposers and executors arrays
+        address[] memory proposers = new address[](1);
+        proposers[0] = proposer;
+
+        address[] memory executors = new address[](1);
+        executors[0] = executor;
+
+        // Deploy MockDiamond first
+        mockDiamond = new MockDiamond();
+
+        // Then deploy timelock with correct mockDiamond address
+        timelock = new LiFiTimelockController(
+            MIN_DELAY,
+            proposers,
+            executors,
+            deployerWallet,
+            admin,
+            address(mockDiamond)
+        );
+
+        // deploy with invalid minDelay
+        vm.expectRevert(InvalidConfig.selector);
+        // Then deploy timelock with correct mockDiamond address
+        timelock = new LiFiTimelockController(
+            0,
+            proposers,
+            executors,
+            deployerWallet,
+            admin,
+            address(mockDiamond)
+        );
+
```

Comment:
there is one missing test case for empty proposers array

Reply:
added